buildSrc/src/main/kotlin/com/datadog/gradle/config/MavenConfig.kt

```diff
@@ -49,7 +49,7 @@ fun Project.publishingConfig(projectDescription: String) {
 
                 groupId = MavenConfig.GROUP_ID
                 artifactId = projectName
-                version = AndroidConfig.VERSION.name
+                version = AndroidConfig.VERSION.name + "-debug"
```

Comment:
I'm afraid that if we publish it, it will be picked by the Gradle still following the ordering rule https://docs.gradle.org/current/userguide/single_versions.html#version_ordering in case of the dynamic versioning use. And given that it has public API change in the `startResource` method, this can be breaking.

How we are going to distribute/publish this version?

Reply:
Actually, according to that documentation 
> A version with an extra non-numeric part is considered lower than a version without: 1.1.a < 1.1

Following that logic, a version `2.11.0-debug` will be considered lower that the existing `2.11.0`, so for the (I'm guessing) few customers who might use dynamic versioning, this still won't be an issue.

Reply:
ok, we are safe on that side then.